projects/kit/components/input-month/native-month-picker/native-month-picker.style.less

```diff
@@ -0,0 +1,17 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+// TODO: add tui-textfield:has([tuiInputMonth])
```

Comment:
tui-textfield input[tuiInputMonth] + .t-content input[type='month']
Maybe like that?